addons/payment/static/src/js/payment_form.js

```diff
@@ -517,10 +517,10 @@ publicWidget.registry.PaymentForm = publicWidget.Widget.extend({
                 // if there's records linked to this payment method
                 var content = '';
                 result[pm_id].forEach(function (sub) {
-                    content += '<p><a href="' + sub.url + '" title="' + sub.description + '">' + sub.name + '</a><p/>';
+                    content += '<p><a href="' + sub.url + '" title="' + sub.description + '">' + sub.name + '</a></p>';
                 });
 
-                content = $('<div>').html(_t('<p>This card is currently linked to the following records:<p/>') + content);
+                content = $('<div>').html('<p>' + _t('This card is currently linked to the following records:') + '</p>' + content);
```

Comment:
need to update pot translation too

seems cleaner as fix, but at the end, why it was not displayed ? because invalide <p> ?

Reply:
Because commit msg let think that you fix the name of token...
But your code is only in a test...

            'name': 'XXXXXXXXXXXX%s - %s' % (data['cc_number'][-4:], data['cc_holder_name'])

Reply:
The actual problem fix is only the change on "payment_acquirer.py": the token of the "Test Payment Acquirer" had no name.
While locating this problem, I encountered some wrongly escaped error messages that were generated by "payment_form.js". Maybe these changes should be in a distinct commit.

Reply:
so the initial bug that has been reported is not fixed ?

Reply:
No no, it is fixed.
The module name is misleading: the token generation is fixed in the model file, but the module itself is named "payment_test". The reported bug itself was about that module: the other payment acquirers do include a name in their generated tokens.